spec/std/array_spec.cr

```diff
@@ -873,6 +873,83 @@ describe "Array" do
     end
   end
 
+  describe "insert_all" do
+    it "inserts with index 0" do
+      a = [2, 3]
+      expected = [0, 1, 2, 3]
+      a.insert_all(0, [0, 1])
+      a.should eq(expected)
```

Comment:
**polish:** The separate variable `expected` isn't really necessary and could be inlined for more conciseness.
```suggestion
      a.should eq([0, 1, 2, 3])
```
Ditto all below.